packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -392,6 +394,20 @@ class _TableRow<T> extends StatefulWidget {
   /// When the value is toggled, this row will animate in or out.
   final bool isShown;
 
+  /// The background color of the row.
+  ///
+  /// If null, defaults to `Theme.of(context).canvasColor`.
+  final Color backgroundColor;
+
+  /// Gets a color to use for this row at a given index.
+  static Color colorFor(BuildContext context, int index) {
+    final theme = Theme.of(context);
+    final alternateColor = theme.brightness == Brightness.dark
```

Comment:
What are your thoughts on writing it this way versus
```dart
final alternateColor = ThemedColor(devtoolsGrey[100], devtoolsGrey[900])
```
?

Reply:
I think we want to base light/dark color selection on theme brightness.  This allows us to, for instance, use Flutter's ThemeMode to respect the system brightness setting when we deploy on desktop.

This could be an interesting, application of extension methods:

```
extension DevToolsTheme on Theme {
  Color selectColor(Color light, Color dark);
}

...

build(context) {
  final themedColor = Theme.of(context).selectColor(Colors.lightBlue, Colors.grey);
}
```

I'm not sure if I like the idea of using extension methods like this.

This is something we can refactor ThemedColor to do, so I'll switch to using that for now and leave an issue regarding this.